index.html

```diff
@@ -0,0 +1,251 @@
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+            
+        </div>
+        <div class="container nav__container">
+            
+            <div class="nav-top">
+                <div class="btn-holder">
+                    <button id="open-menu-btn" class="open-menu-btn">
+                        <i class='bx bx-menu'></i>
+                    </button>
+                    <button id="close-menu-btn" class="close-menu-btn">
+                        <i class='bx bx-x'></i>
+                    </button>
+                </div>
+                <a class="nav-title" href="index.html">
+                    <span>
+                        <img src="./assets/images/trans-logo1.png" alt="mylogo">
+                        <!-- <img src="./assets/images" alt="mylogo"> -->
+                    </span>
+                </a>
+            </div>
+            <div class="nav-bottom">
+                <ul class="nav-menu">
+                    <li> <a  class="nav-item0" href="about.html">About</a></li>
+                    <li> <a class="nav-item1" href="#">Program</a></li>
+                    <li> <a class="nav-item2" href="#">Join</a></li>
+                    <li> <a class="nav-item3" href="#">News</a></li>
+                    <li> <a class="nav-item4" href="#">Sponsor</a></li>
+                    <li> <a class="Campaign" href="#">CC Campaign</a></li>
+                </ul>
+            </div>
+           
+           
+           
+        </div>
+    </nav>
+
+    <header>
+        <article class="container header__container">
+            <article class="header-content">
+                <article class="header-element first">
+                    <h1>"Hello! Coding world"</h1>
+                    <h2 class="header_Ptext">BKE WEB DEVELOPMENT </br> COURSE FOR DEVELOPERS </br> 2023</h2>
+                    
+    
+                </article>
+                <article class="header-element second">
+                    <p>
+                        A joyful celebration believing in the value of openness and sharing,
+                        creating a positive change with people from over 
+                        80 countries is taking place in October, in Korea.
+                        A joyful celebration believing in the value of openness and sharing,
+                        creating a positive change with people from over 
+                        80 countries is taking place in October, in Korea.
+                    </p>
+                </article>
+                <article class="header-element third">
+                    <h3>2023.10.15(THU) ~ 16(FRI)</h3>
+                    <h4>
+                        @ Kenya National Institute of Conference (KICC), at Nairobi.
+                    </h4>
+                </article>
+            </article>
+            
+        </article>
```

Comment:
- You have done well so far, but you have not styled this page according to the design template, I suggest that the background blend for this section should cover the entire section and not just a part of it. Please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

- Also the font-color for the text I highlighted in the image below is not the same as seen in the design template. Please note that, though you are required to personalize the content of the page, you are required to stick to the design layout, background colors, font-style etc. Please fix this 🙏 
Your project:
- Please try to replicate the design template as closely as possible 😄 
<img width="950" alt="yw4w" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/b125e816-83d2-44db-a846-ed4e9e89f673">

Design template:
<img width="627" alt="rwegwrg" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/909712a9-7e0b-42f0-9401-225f9081bc6f">